frontend/src/views/account-mgmt/OrganizationFacilityView.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <v-container fluid>
+    <h1>Update Organization Information</h1>
+    <v-row>
+      <v-col class="ml-6 mt-6 pb-0">
+        <h4>Organization Details</h4>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="ml-6 pt-0">
+        <OrganizationInfo class="mt-1" :editable="true" />
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="ml-6 pt-0 pb-1">
+        <h4>Facilities</h4>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12" class="ml-6 pr-9 pt-0">
+        <v-card class="pa-6" variant="outlined">
+          <v-skeleton-loader :loading="loading" type="table-tbody">
+            <v-row>
+              <v-col cols="11">
+                <v-card v-for="(item, index) in this.facilities"
+                  :to="{ name: 'facility', params: { facilityId: item.facilityId } }"
+                  :key="index"
+                  clickable
```

Comment:
Can we make these cards look a little closer to the home page cards? I don't think we need hover or clickable. And if you replace :to with @click (like in HomeView.vue) you won't get the hyperlink-ish colors.

Also I think the following changes might give the cards a more consistent feel:

.facility-card {
  display: inline-block;
  vertical-align: top;
  min-width: 200px;
  max-width: 300px;
  padding: 8px;
  margin: 4px;
}

Reply:
updated.

Reply:
Can we also remove clickable and hover?

Reply:
oops, missed that. fixed.